recipes/linearmodels/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "linearmodels" %}
+{% set version = "4.17" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 29f07c73be57b5356da41cad83ae53604b215b00cbf3e61347fe5d434cdcfae2
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I don't think this can be noarch if it relies on Cython

Reply:
This was actually a question I had. Couldn't tell from the docs. The cython files themselves aren't OS specific, but obviously compilation happens at some point. If not noarch, what's the appropriate method of restricting to py3k?

Reply:
build: skip: True  # [py <30]

Reply:
I have copied the pandas recipe and just added a build -> skip. Hopefully that is correct. :-)